src/components/GameCard/GameCard.module.css

```diff
@@ -0,0 +1,20 @@
+.card {
```

Comment:
Add cursor: pointer; as the component should navigate to another component

Reply:
done :)